packages/backend/controllers/sessionController.ts

```diff
@@ -37,7 +37,7 @@ class SessionController {
     }
 
     public async updateSession(req: AuthenticatedRequest, res: Response) {
-        const session = await Session.findOneAndUpdate({ _id: req.params.id, user: req.user_id }, {
+        const session = await Session.findByIdAndUpdate(req.params.id, {
             ...req.body
```

Comment:
no need to destructure then put in object again. can follow findByIdAndUpdate for goal